scripts/webpack/prod.servers.config.js

```diff
@@ -1,3 +1,4 @@
+require('./utils/dotenv')
```

Comment:
maybe redundant bc further down we do `
const DOTENV = path.join(PROJECT_ROOT, 'scripts/webpack/utils/dotenv.js')`, but on the upside it's consistent with what we do in `prod.client.config.js`

Reply:
ah, this is probably the fix. i see `getWebpackPublicPath` doesn't import the env vars